datafusion/sqllogictest/src/engines/postgres_engine/mod.rs

```diff
@@ -300,6 +305,15 @@ impl sqllogictest::AsyncDB for Postgres {
     fn engine_name(&self) -> &str {
         "postgres"
     }
+
+    async fn shutdown(&mut self) {
```

Comment:
Similar to https://github.com/risinglightdb/sqllogictest-rs/blob/89d0d3c812c00609100f8b91612440260e0c30f1/sqllogictest-engines/src/postgres.rs#L54

Reply:
This is a great change 👍